tripy/tests/frontend/module/conftest.py

```diff
@@ -139,3 +139,12 @@ def mixed_network():
 @pytest.fixture
 def complex_network():
     yield ComplexNetwork()
+
+@pytest.fixture
+def sequential_network():
+    yield tp.Sequential(tp.Linear(1, 3), tp.Linear(3, 2))
+
+
+@pytest.fixture
+def nested_sequential_network():
+    yield tp.Sequential(tp.Linear(2, 4), tp.Sequential(tp.Linear(4, 3), tp.Linear(3, 1)))
```

Comment:
Could we define this in the test file? `conftest.py` is nice when we want to share fixtures but it otherwise just makes tests harder to read IMO.

Reply:
done